static/js/templates.js

```diff
@@ -69,7 +69,15 @@ Handlebars.registerHelper('tr', function (context, options) {
     //     1. `context` is very important. It can be `this` or an
     //        object or key of the current context.
     //     2. Use `__` instead of `{{` and `}}` to declare expressions
-    const result = i18n.t(options.fn(context), context);
+    let context_string = options.fn(context);
+    const split_strings = context_string.split('\n');
+    context_string = '';
+    for (let i = 0; i < split_strings.length; i += 1) {
+        context_string += split_strings[i].trim();
+        context_string += " ";
+    }
+    context_string = context_string.trim();
```

Comment:
Could be simplified using `map` and `join` (like the corresponding Python):
```js
options.fn(context).trim().split("\n").map(s => s.trim()).join(" ")
```

Reply:
Thanks! That looks nice!